installer/default_platform.conf

```diff
@@ -1,119 +1,5 @@
 #!/bin/sh
```

Comment:
It is not ideal that we have .conf file with so complex bash code. And there are many variables need replacement, like `%%SONIC_ROOT%%`.